### ISSUE TITLE
[GDNative] Node properties are not loaded from scene file.

### ISSUE THREAD

**Issue by bruvzg** [godotengine/godot#18088]
**Godot version:**

master ea0e73f3c

**Issue description:**

Properties of GDNative Node (registered from editor plugin, and added to scene) are not loaded from scene file correctly.

If property record in `.tscn` file is located before `script = ...` (editor saves scene like this) it is not set correctly when scene is loaded.
If property is manually moved to be after `script = ...` it is loaded correctly (until scene is saved in editor next time).

*Property value is NOT loaded:*
```ini
[node name="TestNode" type="Node" parent="." index="0"]
test_value = 800
script = ExtResource( 1 )
__meta__ = {
"_editor_icon": ExtResource( 2 )
}
```

*Property value IS loaded:*
```ini
[node name="TestNode" type="Node" parent="." index="0"]
script = ExtResource( 1 )
test_value = 800
__meta__ = {
"_editor_icon": ExtResource( 2 )
}
```

**Steps to reproduce:**

Add GDNative Node to scene.
Set property value in editor.
Save project.
Re-open project in editor (Property value is not set to saved value).

**Minimal reproduction project:**

[TestProject.zip](https://github.com/godotengine/godot/files/1890655/TestProject.zip)
Includes: project, plugin, gdnative source, binary libs (macOS, Windows 64-bit, Linux 64-bit).

**Comment by sheepandshepherd** [comment]
Related to #17672. I think `NativeScriptInstance::get_property_list` just needs to sort script properties to after `script` like GDScript does, but I still can't figure out how exactly this is done for GDScript...

**Comment by bruvzg** [comment]
Fixed via #18119.
